windowsforms/Conway's-Game-of-Life/VB/Conway's-Game-of-Life/Conway's-Game-of-Life.vbproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Sudoku</RootNamespace>
```

Comment:
Remove the redundant RootNamespace

Reply:
It should have been Conway's-Game-of-Life, but I can delete it.